README.md

```diff
@@ -82,5 +82,17 @@ Options:
 bash-3.2$
 ```
 
+### Autorest plugin configuration
+- Please don't edit this section unless you're re-configuring how oav plugs in to AutoRest
+AutoRest needs the below config to pick this up as a plug-in - see https://github.com/Azure/autorest/blob/master/docs/developer/architecture/AutoRest-extension.md
+
+``` yaml $(model-validator)
+pipeline:
+  swagger-document/model-validator:
+    input: swagger-document/identity
+    scope: azure-validator-composed
```

Comment:
think you can get rid of that - even more so, I don't see a scope called `azure-validator-composed` in here, that was necessary for Deepak distinction between rules for single vs. merged Swagger - model validator doesn't have such distinction

Reply:
@veronicagg any plans on moving forward with this?